### ISSUE TITLE
[Feature request] Don't hide the tab favicon of a collapsed branch

### ISSUE THREAD

**Issue by Elaws** [mbnuqw/sidebery#794]
## Description

When a branch is collapsed in a tabs tree, the favicon of the top tab is hidden by an arrow icon :

![0_sidebery](https://user-images.githubusercontent.com/52013479/194756387-2ae38cd8-11c8-41e1-b39e-7c74d0a20075.png)

The problem is that it makes the tabs tree much less readable : favicons are great visual indicators and should not be hidden. Something like that (or really anything that **does not** hide the favicon but still shows it's a collapsed branch) would be much better :

![0_sidebery3](https://user-images.githubusercontent.com/52013479/194756577-3a16c5fd-e54d-42d7-b07c-9f3859875d84.png)

Thanks !

**Comment by emvaized** [comment]
You can try this CSS to show arrow only on hover:

```
/* Show expand arrow only on hover */
.Tab[data-parent="true"][data-folded="true"]:not(:hover) .fav .exp {
 opacity: 0;
}

/* Shift child count for better visibility */
.Tab .fav > .child-count {
 right: -4px;
}
```

**Comment by Elaws** [comment]
Thank you @emvaized, this does the trick !

**Comment by emvaized** [comment]
@Elaws 
As an alternative, you can also try something like this – just shift the arrow and child count to the corner. 
But it may require some playing with numbers to make it look good in your setup. 

<img width="266" alt="screenshot" src="https://user-images.githubusercontent.com/37851576/194939412-fd69d853-b25a-4bc3-9fa4-94e4d2c95122.png">

```
/* Increase opacity for folded tab icon */
.Tab[data-parent="true"][data-folded="true"]:not(:hover) .fav img {
	opacity: 1;
}

/* Shift the arrow */
.Tab[data-parent="true"] .fav .exp {
	top: 55%;
	left: -12px;
        opacity: 1 !important; /* <- keeps arrow always visible */
}

/* Shift the child count */
.Tab .fav > .child-count {
	right: 0px;
	top: 105%;
	bottom: unset;
	left: 4.5px;
}
```
<br/>

It also looks great with "use lines instead of dots" snippet, which I recently wrote for myself:
![Screenshot_11-10-22_01-57](https://user-images.githubusercontent.com/37851576/194964384-04d703fa-555b-4535-994c-f4d93a8dd86d.png)

```
/* Show tab indents as lines */
.Tab .body::after {
	top: 0;
	height: 100%;
	border-radius: 0;
	width: 1px;
	left: 7.5px;
}
```

**Comment by Elaws** [comment]
@emvaized : Great css snipets, I'll use them : thanks !

**Comment by Elaws** [comment]
@emvaized : I tried your `Show tabs indents as lines` snippet but lines do not appear. Are there options to activate in Sidebery ?

```
/* Show tab indents as lines */
.Tab .body::after {
	top: 0;
	height: 100%;
	border-radius: 0;
	width: 1px;
	left: 7.5px;
}
```

**Comment by emvaized** [comment]
@Elaws Yes, you have to enable `Show marks to indicate tabs sub-tree levels` to show default dots

**Comment by Elaws** [comment]
@emvaized : Weird, it's already activated, and all it shows is dots, not lines.

```
/* Increase opacity for folded tab icon */
.Tab[data-parent="true"][data-folded="true"]:not(:hover) .fav img {
	opacity: 1;
}

/* Shift the arrow */
.Tab[data-parent="true"] .fav .exp {
	top: 55%;
	left: -12px;
        opacity: 1 !important; /* <- keeps arrow always visible */
}

/* Shift the child count */
.Tab .fav > .child-count {
	right: 0px;
	top: 105%;
	bottom: unset;
	left: 4.5px;
}

/* Show tab indents as lines */
.Tab .body::after {
	top: 0;
	height: 100%;
	border-radius: 0;
	width: 1px;
	left: 7.5px;
}
```

**Comment by emvaized** [comment]
@Elaws 
It's important to note that I'm using version 5, and the snippet was written for that version. If you're on v4, it may be needed to inspect the tree and rewrite some things to make it work

**Comment by mbnuqw** [comment]
@emvaized, thanks for the snippets and @Elaws, thanks for the feature request. I'll add an option to choose the style of arrow icon (when it will be shown and where).

**Comment by Qwerty-Space** [comment]
I've implemented this like so:
![image](https://github.com/mbnuqw/sidebery/assets/6967592/d0e6b829-a77c-4722-8b21-5afdd918bfdf)

```css
/* Resize the favicon div */
.Tab[data-parent="true"] .fav{
	margin-left: 8%;
}

/* Shift the arrow */
.Tab[data-parent="true"] .fav .exp {
	margin: auto;
	left: -20px;
        opacity: 1 !important; /* <- keeps arrow always visible */
}

/* Shift the child count */
.Tab .fav > .child-count {
	top: 78%;
	left: -100%;
}
```

**Comment by waldyrious** [comment]
@Elaws, could you or @mbnuqw change the title of this issue to say "favicon" instead of "thumbnail"? Otherwse it might be confused with the "Tabs preview" feature 😅
